x/lend/client/cli/tx.go

```diff
@@ -785,6 +785,78 @@ func CmdAddAssetToPairProposal() *cobra.Command {
 	return cmd
 }
 
+func CmdAddMultipleAssetToPairProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add-multiple-asset-to-pair-mapping [asset_id] [pool_id] [pair_id] ",
```

Comment:
if the input is `,` separated values, please use  plurals for the given variable names